packages/integration-tests/projects/suite-web/tests/recovery/t2-dry-run-persistence.test.ts

```diff
@@ -13,7 +13,7 @@ describe('Recovery - dry run', () => {
         cy.viewport(1024, 768).resetDb();
     });
 
-    it('Communication between device and application is automatically established whenever app detects device in recovery mode', () => {
+    it.skip('Communication between device and application is automatically established whenever app detects device in recovery mode', () => {
```

Comment:
Please add a comment here linking to the issue where we have discussed the problems (even if it is closed)